test/it/code-samples/negative/public-$w-non-string-selector/core.js

```diff
@@ -0,0 +1,8 @@
+const arr = ["firstSelector", "secondSelector"]
+const obj = {
+    test: "testSelector"
+}
+$w(arr).anyProperty();
+$w(obj).z()
+$w(8).testo()
+$w(true).another()
```

Comment:
i think we should start adding comments that explain what the test checks.
some general description + maybe comments for lines that check specific cases

otherwise, it's really difficult to understand.

Reply:
will add comments